.github/workflows/libjs-test262.yml

```diff
@@ -133,3 +133,38 @@ jobs:
         continue-on-error: true
         working-directory: libjs-test262
         run: ./per_file_result_diff.py -o per-file-bytecode-master.json -n ../libjs-website/test262/data/per-file-bytecode-master.json
+
+      - name: Build js
+        working-directory: libjs-test262/Build
+        run: |
+          ninja js
+
+      - name: Create js package
+        working-directory: libjs-test262/Build
+        run: |
+          mkdir -p pkg/bin pkg/lib
+
+          cp --preserve _deps/lagom-build/js pkg/bin
```

Comment:
I'm not super thrilled about packaging js from the build directory, since, like Ali said that's what's mandating the LD_LIBRARY_PATH hack. 

Not sure if we need to do this now or as a yak for another day but I think a cleaner impl would be to do one of:

1) do an install and package from there
2) create a special exports target to assemble this package in Lagom/CMakeLists.txt how we like using CMake components
3) use `patchelf` (https://github.com/NixOS/patchelf) which has binary packages on most distros to `patchelf --set-rpath` on the libs/binaries to match this packaging layout.

Reply:
This should be working better now - Updated Lagom/CMakeLists.txt to include some packaging rules, and now CI just runs a `cpack` command to create it without any `LD_LIBRARY_PATH` shenanigans.

Ideally we could run just `ninja package` instead of `cpack` in CI, but I couldn't get it to work without specifying where Lagom is built for test262 by overriding `CPACK_INSTALL_CMAKE_PROJECTS`.

Reply:
And here's a build with those changes: https://github.com/trflynn89/serenity/actions/runs/1813691949